src/OrchardCore.Modules/OrchardCore.Media.AmazonS3/Helpers/OptionsFluidParserHelper.cs

```diff
@@ -0,0 +1,45 @@
+using Fluid;
+using OrchardCore.Environment.Shell;
+
+namespace OrchardCore.Media.AmazonS3.Helpers;
+
+// This is almost the same as in OrchardCore.Media.Azure but there isn't really a good common place for it.
```

Comment:
This is meant as a warning so if anybody touches this in the future, they might have a better place to put it with the other class in a DRY way (like a `Core` project, what I wouldn't want to introduce for just this class).

Reply:
Let's suppose that in 3 months from now, somebody finds a bug in this class and fixes it. Since the other class will have the same bug, it'd be best if they'd fix it there too. However, at that point, we won't remember this discussion. Wouldn't it help to somehow make it apparent that the other class exists as well?

Ideally, we wouldn't need this and the whole code would be DRY but since it's not, I figure at least a comment would help. Maybe?

```c#
// This is almost the same as in OrchardCore.Media.Azure. If you find any issue here, fix it in the other class too.
```